pom.xml

```diff
@@ -325,6 +325,24 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mortbay.jetty</groupId>
```

Comment:
It's a little strange that further up in the pom we exclude this dependency from `hadoop-common` and `hadoop-hdfs` but then introduce it as a top level dependency.

I assume it's needed by `hadoop-azure` but if that's the case why isn't `hadoop-azure` pulling it in as a transitive dependency? I looked at the pom for `hadoop-azure` https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-azure/2.7.3/hadoop-azure-2.7.3.pom but couldn't find `jetty-util` as a dependency. Can you explain why this is needed?